Readme.md

```diff
@@ -16,7 +16,13 @@ Note they are unsupported and we strongly recommend you to upgrade your PHP vers
 # Installation
 
 All versions can be found in the [releases list](https://github.com/PrestaShop/autoupgrade/releases).
-Zip archives can be directly uploaded on your module page.
+
+**You need to follow a few steps to create a valid module package**
+1. Unzip the release file
```

Comment:
These steps are valid if you chose the "Source code" option and should be specified.
I forgot it for the version 4.4.1, but I usually upload an additional zip file with a ready to use module.